### ISSUE TITLE
RichTextBlock with inline hyperlink fails Accessibility Insights Tool test when hyperlink isn't visible

### ISSUE THREAD

**Issue by ppsz** [microsoft/microsoft-ui-xaml#10976]
### Describe the bug

I have a RichTextBlock with hyperlink inside a ScrollViewer. The IsOffscreen property of that hyperlink isn't updated properly when the hyperlink is out of view.
It makes Accessibility Insights Tool fail with "An on-screen element must not have a null BoundingRectangle property."

### Why is this important?

This issue may cause a confusion for users depending on this accessibility feature when there's a hyperlink in the RichTextBlock content

### Steps to reproduce the bug

[TextBlockHyperlinkIssue.zip](https://github.com/user-attachments/files/24909085/TextBlockHyperlinkIssue.zip)

1a. Make a new project using WinUI template and add following structure, for example in MainWindow.xaml:
```
<ScrollView>
    <RichTextBlock>
        <Paragraph>
        Text long enough to hide the hyperlink below ...
            <Hyperlink>some link</Hyperlink>
        </Paragraph>
    </RichTextBlock>
</ScrollView>
```
1b. Use attached example project
2. Start the app
3. Make sure the hyperlink isn't visible in the app window
4. Start Accessibility Insights Tool
5. Run automated checks on the app

### Actual behavior

Automated check finds a failure on the hyperlink: "An on-screen element must not have a null BoundingRectangle property."

### Expected behavior

Accessibility test shouldn't fail

### Screenshots

Failing, hyperlink not visible:
<img width="1429" height="462" alt="Image" src="https://github.com/user-attachments/assets/c29650ac-a451-431c-b856-a5a585b96dad" />

Passing, hyperlink visible:
<img width="1427" height="462" alt="Image" src="https://github.com/user-attachments/assets/13a67535-acea-4482-94e7-abfdb6b4bd21" />

### NuGet package version

WinUI 3 - Windows App SDK 1.8.3: 1.8.251106002

### Windows version

Windows 11 (24H2): Build 26100

**Comment by godlytalias** [comment]
Fixed with https://github.com/microsoft/microsoft-ui-xaml/commit/184065ab5f453bf9ae1c6c8b684a8fca1421dcdc
